query-projector/src/lib.rs

```diff
@@ -116,6 +116,29 @@ impl QueryResults {
             &FindRel(_)    => Box::new(|| QueryResults::Rel(vec![])),
         }
     }
+
+    pub fn to_pretty(&self) -> Result<String> {
+        use QueryResults::*;
+        match self {
+            &Scalar(Some(ref val)) => { 
+                Ok(val.value_type().to_edn_value().to_pretty(120).unwrap_or(String::new()) )
```

Comment:
You don't want to ignore the errors; you want to funnel them up into the outer `Result`.    You want something like
```rust
val.value_type().to_edn_value().to_pretty(120)
```
possibly with a `.map_err()` to make the `Result` types compatible.